src/Components/TourGuideCard/TourGuideCard.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import './tourGuideCard.css';
+import Button from '../Button/Button';
+import { Card, Row, Col, Tooltip, Tag, Skeleton } from 'antd';
+import { EllipsisOutlined } from '@ant-design/icons';
+
+function TourGuideCard({ props }) {
```

Comment:
change this to a better name than `props`
```suggestion
function TourGuideCard({ tourGuide }) {
```